### PR TITLE
Update unrestrict-code-execution.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated response when you send multiple API calls to the same device to remove app restriction

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5906

File: microsoft-365/security/defender-endpoint/unrestrict-code-execution.md [hunk 1]
```diff
@@ -89,6 +89,8 @@ Comment|String|Comment to associate with the action. **Required**.
 
 If successful, this method returns 201 - Created response code and [Machine Action](machineaction.md) in the response body.
 
+If you send multiple API calls to to remove app restriction for the same device, it returns "pending machine action" or HTTP 400 with the message "Action is already in progress".
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/unrestrict-code-execution.md]:
```suggestion
If you send multiple API calls to remove app restrictions for the same device, it returns "pending machine action" or HTTP 400 with the message "Action is already in progress".
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @msbemba, kindly commit the suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**